### PR TITLE
[#1808] Use long month names consistently in ssd templates

### PR DESCRIPTION
**Pull request by pi-sigma**
Taiga: [#1808](https://taiga.maykinmedia.nl/project/open-inwoner/issue/1808)

- set empty title in PDF templates to prevent duplication of report name in tab
- use long month name in case no reports are found

### CODE REVIEW

File: src/open_inwoner/ssd/templates/jaaropgave.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 <!DOCTYPE html>
 <html lang="nl">
+<head>
+  <title></title>
+</head>
```
Comment (stevenbal) [inline src/open_inwoner/ssd/templates/jaaropgave.html:7]: Why was this added?

File: src/open_inwoner/ssd/tests/test_views.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def test_uitkering_post_fail(self, mock_report):
         self.assertEqual(response.status_code, HTTPStatus.OK)
 
         self.assertContains(
-            response, "Geen uitkeringsspecificatie gevonden voor Dec 1985"
+            response, "Geen uitkeringsspecificatie gevonden voor december 1985"
```
Comment (stevenbal) [inline src/open_inwoner/ssd/tests/test_views.py:90]: In the taiga issue I see that the title of the `Maandspecificatie` pdf is what should be fixed, so I would expect that to be addressed too?

**Review by stevenbal — CHANGES_REQUESTED** [review]

File: src/open_inwoner/ssd/tests/test_views.py (hunk 2 above)
Comment (pi-sigma) [reply src/open_inwoner/ssd/tests/test_views.py:90]: This was resolved last year, see [https://github.com/maykinmedia/open-inwoner/blob/fix/1808-maandspecificatie-header/src/open_inwoner/ssd/client.py#L192](https://github.com/maykinmedia/open-inwoner/blob/fix/1808-maandspecificatie-header/src/open_inwoner/ssd/client.py#L192)

File: src/open_inwoner/ssd/templates/jaaropgave.html (hunk 1 above)
Comment (pi-sigma) [reply src/open_inwoner/ssd/templates/jaaropgave.html:7]: The title is duplicated in case a new tab is opened for file download (at least on Firefox). Apparently it is not (yet) standardized what browsers do with the title. The `<title><title>` effectively re-sets whatever the browser does with the title, afterwards we define our custom title. It's a hack, but I don't know a better way to deal with this. Perhaps @jiromaykin has some idea about this. I did however remove the extra `head`, which was an oversight.

Comment (pi-sigma) [reply src/open_inwoner/ssd/templates/jaaropgave.html]: Discussed with Jiro, I'll revert changes and create a separate issue for this because it's an accessibility issue.

**Comment by stevenbal** [comment]
@pi-sigma is this PR ready for re-review?

**Comment by pi-sigma** [comment]
@ Steven Yes, this one is ready, forgot to re-request.
